Python/Aws_EC2/create_ec2_instance.py

```diff
@@ -0,0 +1,46 @@
+import boto3
+
+
+class ec2_instance:
```

Comment:
The Class Names, COnstant Names and Function names don't follow the proper naming convention.

These will help you:

[Styling Guide](https://github.com/HarshCasper/Rotten-Scripts/blob/master/STYLE_GUIDELINES.md/#Python)
[Official PEP-8 Guide](https://www.python.org/dev/peps/pep-0008/)